chapter_convolutional-modern/vgg_vn.md

```diff
@@ -212,7 +215,10 @@ d2l.train_ch6(net, train_iter, test_iter, num_epochs, lr)
 4. Refer to Table 1 in :cite:`Simonyan.Zisserman.2014` to construct other common models, such as VGG-16 or VGG-19.
 -->
 
-*dịch đoạn phía trên*
+1. Khi in ra kích thước của các lớp, chúng ta chỉ thấy 8 kết quả chứ không phải 11. Thông tin về 3 lớp còn lại nằm ở đâu?
```

Comment:
1. Khi in ra kích thước của các `tầng`, chúng ta chỉ thấy 8 kết quả chứ không phải 11. Thông tin về 3 `tầng` còn lại nằm ở đâu?